### PR TITLE
feat(upgrades): upgrade libraries, migrate to reanimated and new feature with gesture-handler

### PR DESCRIPTION
**Pull request by frodriguez-hu**
- Upgrading to the last library config based on builder-bob.
- Migrate animations to reanimated
- Create Gesture Handler animation (optional)

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+            message: 'Please use @services/encryptedStorage instead',
+          },
+          {
+            name: 'react-native',
+            importNames: ['ActivityIndicator'],
+            message: 'Please use @components/ActivityIndicator instead',
+          },
+        ],
+      },
+    ],
```
Comment (sfernandez11) [inline .eslintrc.js]: Delete this code!

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+          ['sibling', 'index'],
+        ],
+        'pathGroups': [
+          {
+            pattern:
+              '@[assets|components|config|hooks|interfaces|modules|navigation|redux|services|shared]*/**',
+            group: 'internal',
+            position: 'after',
+          },
+        ],
```
Comment (sfernandez11) [inline .eslintrc.js]: And this!

**Review by sfernandez11 — APPROVED** [review]
Nice work Feli!

File: .eslintrc.js (hunk 2 above)
Comment (felire) [reply .eslintrc.js]: Thanks!

File: .eslintrc.js (hunk 1 above)
Comment (felire) [reply .eslintrc.js]: Great! Done!

**Pull request merged**